apps/desktop/src/components/views/terminal/terminal-panel.tsx

```
@@ -0,0 +1,430 @@
+              onKeyDown={handleNameKeyDown}
+              onBlur={finishNameEditing}
+              onClick={(e) => e.stopPropagation()}
+              placeholder={shellName}
+              className="w-24 rounded border border-border bg-background px-1 py-0 text-xs focus:outline-none focus:ring-1 focus:ring-primary"
+            />
+          ) : (
+            <span
+              className="cursor-pointer truncate text-xs text-foreground hover:text-primary"
+              onDoubleClick={handleNameDoubleClick}
+              title="Double-click to rename"
+            >
```

Biome (2.3.13)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)